framework/source/class/qx/io/remote/Rpc.js

```diff
@@ -390,7 +390,7 @@ qx.Class.define("qx.io.remote.Rpc",
      */
     protocol :
     {
-      init : "qx1",
+      init : "2.0",
```

Comment:
I'm not using it, but this is a BC breaking change. Or did I misunderstand something?

Reply:
It's a breaking change but can be easily made BC by setting the property. I don't think many people are using the legacy mode. We can put it into the relase notes under "breaking changes".

Reply:
has there been any other mode than the qx1 mode ?

Reply:
all our apps use qx jsonrpc